generator/tests/test_strfunc_code/run_strfunc_tests.py

```diff
@@ -205,7 +225,7 @@ def main():
     fail += execute_tests(sf.plural_no_prefix, data, fails)
     gv.is_package = False
     gv.prefix = "Rosie"
-    data = {"Rosiefox": "foxes", "Rosiechild": "children", "RosieCat": "Cats"}
+    data = {"RosieFox": "Foxes", "RosieChild": "Children", "RosieCat": "Cats"}
```

Comment:
I think I probably had one of my tests all lower-case to check it would be handled ok.